ERCS/eip-draft-delegation-framework.md

```diff
@@ -0,0 +1,178 @@
+---
```

Comment:
```suggestion
---
eip: 7710
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.